lua/lspconfig/util.lua

```diff
@@ -3,6 +3,7 @@ local validate = vim.validate
 local api = vim.api
 local lsp = vim.lsp
 local uv = vim.loop
+local nvim_nine_above = vim.verison().minor > 9
```

Comment:
lspconfig does not need to support 0.9.

also, use `has('nvim-0.9')`. checking only the minor version is fragile

Reply:
> lspconfig does not need to support 0.9.

take some days. Some package managers have not been updated yet